crates/loans/src/mock.rs

```diff
@@ -232,7 +228,10 @@ pub fn with_price(
             (_, currency) if currency == DEFAULT_WRAPPED_CURRENCY => {
                 return MockResult::Return(Ok(Amount::new(amount.amount(), DEFAULT_WRAPPED_CURRENCY)));
             }
-            (currency, _) if currency == DEFAULT_WRAPPED_CURRENCY => {
+            (currency, x) if currency == DEFAULT_WRAPPED_CURRENCY => {
+                return MockResult::Return(Ok(Amount::new(amount.amount(), x)));
+            }
+            (a, b) if a == b => {
```

Comment:
It's needed because this identity conversion is common here: https://github.com/interlay/interbtc/blob/1a1afa90228f37c9ade4acbda8275c2f5cfe85ce/crates/currency/src/lib.rs#L67

E.g. `Amount::new(1, KSM).convert_to(LEND_KSM)` hits this case